frontend/tests/util/isolation.ts

```diff
@@ -94,11 +94,20 @@ export const test = base.extend<CustomTestFixtures, CustomWorkerFixtures>({
 });
 
 const runTobiraCommand = async (tobira: TobiraProcess, args: string[]) => {
-    await new Promise(resolve => {
+    await new Promise((resolve, reject) => {
```

Comment:
`await new Promise` is kind redundant and causes some overhead. You could just `return`. 🤔